components/local-app/main.go

```diff
@@ -261,6 +261,7 @@ func tryConnectToServer(gitpodUrl string, tkn string, reconnectionHandler func()
 		Log:                 logrus.NewEntry(logrus.StandardLogger()),
 		ReconnectionHandler: reconnectionHandler,
 		CloseHandler:        closeHandler,
+		ComponentName:       "local-companion",
```

Comment:
@csweichel @geropl I think it would make sense to provide a version as well. WDYT? 

With supervisor it is probably not so actual, but there can be many different versions of the local app so it could be useful to see in our logs which was used to connect in our logs instead of asking the user.

i.e. it can be either `User-Agent: gitpod/local-companion 12345`, or we add another headers like `X-Client-Version: 12345`  wdyt?

Reply:
ok, let's add a separate `X-Client-Version`